Source/Library/UIColor+Hex.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+public extension UIColor {
+    
+    convenience init(hex string: String) {
```

Comment:
@RamonGilabert good point, one could argue that it shouldn't use `hex` but you can apply it in your project when you configure it. That might be the best option here.